src/format/opamFile.ml

```diff
@@ -76,6 +76,7 @@ end
 
 module type IO_Arg = sig
   val internal : string
+  val atomic : bool
```

Comment:
Is there a downside to always having all file writes as atomic?

Reply:
Atomic takes few more steps: first write on tmp, then rename on final location, instead of directly writing on final location.
related #5473

Reply:
Yes but that's fine. I don't think i'm able to see any usecase where this isn't better